backends/utils.py

```diff
@@ -0,0 +1,50 @@
+"""
+Utility functions for the backends
+"""
+from datetime import datetime, timedelta
+
+from requests.exceptions import HTTPError
+from social.apps.django_app.utils import load_strategy
+
+
+class InvalidCredentialStored(Exception):
+    """Custom exception to throw in some specific situations"""
+    def __init__(self, message, http_status_code):
+        super(InvalidCredentialStored, self).__init__(message)
+        self.http_status_code = http_status_code
+
+
+def _refresh_timestamp(user_social):
+    """
+    Private function that refresh an user access token
```

Comment:
this is a bit confusing. would `_send_refresh_request` be a better name for this method? the docstring says this is here to "refresh an user access token" but there's already a method called `refresh_user_token`. i feel like more specific names for these methods would help